cql/ManageRareAbnormality.cql

```diff
@@ -1635,11 +1807,15 @@ define RecommendationForSurveillanceAfterAbnormalities:
           '*Refer to ASCCP Consensus Guideline 2019 Section J.2'
         }
       }
+      
+// =============================================================================      
     // J.3 Guidance for Long-Term Follow-up After Treatment for High-Grade Histology or Cytology
     when ( // J3.1
       HasTreatmentForHighGradeHistologyOrCytology and
       TreatmentForHighGradeHistologyOrCytologyDate >= Now() - 25 years and
-      not InitialIntensiveSurveillancePeriod
+      not InitialIntensiveSurveillancePeriod // AFTER the treatment for high-grade HistologyOrCytology
+      // AND NOT Cervical Histology result AFTER the 'treatment for high-grade HistologyOrCytology
+      // Update aug 28 - sharon assigned task to change this
```

Comment:
Did we intend to commit this? If so, can we mark it as "TODO: *description of pending update*", similarly to how we do elsewhere in the CQL code?

Reply:
Will add update to check for histology after treatment

Reply:
I saw in commit https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/118/commits/c2704894cf2fdb29944fdfe2ff110d3d4094bda7 that the comment was removed, and that you added `and not HasHistologyAfterTreatmentForHighGradeResult`. I still have some questions about the reasoning for adding that line to the L2, but that is more of a question for the clinical team. I documented this question here: [L2 document comment](https://mitre.sharepoint.com/:w:/r/sites/CervicalCancerCDS/Shared%20Documents/CDS%20Development/L2%20Artifacts/Working%20Documents/CDC_CC_CDS_L2Development_v7.docx?d=wc90d0228d64f420bb1565fb44a712630&csf=1&web=1&e=NBNky1&nav=eyJjIjo2ODc4MjMwNjB9). Since this now matches what is in the L2 document, and this PR is not intended to fix section J, I will resolve this comment.